### ISSUE TITLE
When scanning a large number of websites, how can I automatically filter out some similar responses?

### ISSUE THREAD

**Issue by nbxiglk0** [maurosoria/dirsearch#1242]
### What is the question?
Hi,
I would like to ask how to filter out some similar responses when scanning some websites in bulk, I have encountered this situation where all paths return the same status code whether they exist or not, for example 200 or all response will 302 to a same url), and the response packet size also has some slight differences depending on the requested path (often because the input url is also fed into the response), can we request a random path that must not exist before we start scanning and use its response as a template for filtering?

**Comment by nbxiglk0** [comment]
Hi,thanks for your advance,I tried this command and it seemed to work.  
but I ran into another situation where only a few sites in a URL list would have a rate limit, and these sites would start out fine, and then after a while the rate limit would cause all subsequent URL requests to the site to return a similar response (like 403), at which point I had to manually cancel the command and add the size of the response back in. I can't always watch the scanning situation, and I can't be the first to notice this scenario happening, and when I do, I often have already sent a lot of invalid requests, and this time a lot of invalid output tends to fill up the whole screen.
what I'm not sure if there is a way to automatically skip the site when this happens on the way to the scan?
